### PR TITLE
Iets

### PR DESCRIPTION
**Pull request by prinshertog**

### CODE REVIEW

**Review by MathijsDeWilde — CHANGES_REQUESTED** [review]
Please have an actual sentence

**Comment by prinshertog** [comment]
> Please have an actual sentence

Nahhh

File: README.md [hunk 1]
```diff
@@ -36,3 +36,5 @@ npm run build
 You can preview the production build with `npm run preview`.
 
 > To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+
+Iets
```
Comment (MathijsDeWilde) [inline README.md:40]: Please just add our names to the top of the readme in comment

**Review by MathijsDeWilde — CHANGES_REQUESTED** [review]

**Review by MathijsDeWilde — APPROVED** [review]
Well done Jan

**Pull request merged**